setup.py

```diff
@@ -63,7 +70,7 @@
       test_suite='test',
       extras_require={
           'secure': [
-              'pyOpenSSL>=0.14',
+              PYOPENSSL_VERSION,
```

Comment:
This may require a change in setup.cfg too for the wheel

Reply:
Actually it does: https://github.com/urllib3/urllib3/blob/30e9855307aaa9f9f09651f2bbddeb7350eed666/setup.cfg#L14 Your logic here will need to become more complicated.

Reply:
Dang, reasons we can't have nice things. :P Good catch!